bundles/org.openhab.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/internal/PersistenceManagerImpl.java

```diff
@@ -51,13 +52,15 @@
 import org.osgi.service.component.annotations.Reference;
 import org.osgi.service.component.annotations.ReferenceCardinality;
 import org.osgi.service.component.annotations.ReferencePolicy;
+import org.quartz.Job;
```

Comment:
Those imports were added because of a reference in the JavaDoc for the `removeTimers()` method. As we are not using Quartz anymore we should fix the JavaDoc parts.

https://github.com/openhab/openhab-core/blob/ca1cda3f7521d56b85c4ee87a7417ab5aeb4e6f4/bundles/org.openhab.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/internal/PersistenceManagerImpl.java#L360-L374

Reply:
full ack

Reply:
See #852